src/cards/ScenarioNode/ScenarioNode.js

```diff
@@ -376,7 +370,7 @@ ScenarioNode.propTypes = {
     failed: PropTypes.string.isRequired,
     created: PropTypes.string.isRequired,
     delete: PropTypes.string.isRequired,
-    redirect: PropTypes.string.isRequired,
+    edit: PropTypes.string.isRequired,
```

Comment:
I'm not sure but introducing new required prop can be a breaking change, no? Can we just add it as not required?

Reply:
It is inside the labels object and there is also default value for this attribute. I don't really master the behavior of this type of object props with attributes and I don't know why all others labels attributes are required.

Reply:
Actually, if `labels` is not required, adding a new label inside it could be a non-breaking change, provided that we add a fallback default value that can be "patched" inside the previously used prop values.
This would only work for the default language though, translations would be broken, but only for the new feature.

I think this branch will be a breaking change, so we can change these labels anyway :smile: